airbyte-protocol/protocol-models/src/main/java/io/airbyte/protocol/models/JsonSchemaType.java

```diff
@@ -23,12 +23,16 @@ public class JsonSchemaType {
   public static final String CONTENT_ENCODING = "contentEncoding";
   public static final String BASE_64 = "base64";
   public static final String AIRBYTE_TYPE = "airbyte_type";
+  public static final String ITEMS = "items";
+  public static final String STRING_TYPE = "string";
+
 
   public static final JsonSchemaType STRING = JsonSchemaType.builder(JsonSchemaPrimitive.STRING).build();
   public static final JsonSchemaType NUMBER = JsonSchemaType.builder(JsonSchemaPrimitive.NUMBER).build();
   public static final JsonSchemaType INTEGER = JsonSchemaType.builder(JsonSchemaPrimitive.NUMBER).withAirbyteType(AIRYBTE_INT_TYPE).build();
   public static final JsonSchemaType BOOLEAN = JsonSchemaType.builder(JsonSchemaPrimitive.BOOLEAN).build();
   public static final JsonSchemaType OBJECT = JsonSchemaType.builder(JsonSchemaPrimitive.OBJECT).build();
+  public static final JsonSchemaType ARRAY_WITH_STRING_ITEMS = JsonSchemaType.builder(JsonSchemaPrimitive.ARRAY).withItems(STRING_TYPE).build();
```

Comment:
why do we have a constant for an array that contains items of a certain type?

Reply:
Because previously we just have Array without items and it was the reason of [failing ](https://github.com/airbytehq/oncall/issues/351#issuecomment-1190688913) during refresh schema. Currently throwing exception in such case was changed to logging error message and we are adding items to array on source side.

Reply:
Sorry. This really doesn't make sense to me. It seems like for Postgres that we are making the assumption that all arrays will be string arrays. That is not how postgres works. The [docs](https://www.postgresql.org/docs/current/arrays.html) say other types are supported. This is just going to break.

@grishick